src/Microsoft.DotNet.Web.ProjectTemplates/content/StarterWeb-CSharp/Views/Shared/_CookieConsentPartial.cshtml

```diff
@@ -0,0 +1,65 @@
+﻿@using Microsoft.Extensions.Options
+@using Microsoft.AspNetCore.Builder
+@using Microsoft.AspNetCore.Http.Features
+@using Microsoft.Net.Http.Headers
+
+@inject IOptions<CookiePolicyOptions> CookiePolicy
+
+@{
+    var showBanner = (!Context.Features.Get<ITrackingConsentFeature>()?.CanTrack ?? false) && CookiePolicy.Value.ConsentCookie != null;
```

Comment:
What's the actual logic here? what does CookiePolicyOptions.Value.ConsentCookie != null mean?